tutorials/11-probabilistic-pca/11_probabilistic-pca.jmd

```diff
@@ -63,7 +63,7 @@ We use PCA to explore underlying structure or pattern which may not necessarily
 
 #### Step 1: configuration of dependencies
 
-First, we define the dependencies used.
+First, we import the dependencies used.
```

Comment:
```suggestion
First, we load the dependencies used.
```

I think "import" can get confused with `import ...` from Julia and we are instead using `using ...`